enhancements/workload-partitioning/wide-availability-workload-partitioning.md

```diff
@@ -0,0 +1,731 @@
+   `Infrastructure.Status.CPUPartitioning: AllNodes`
+3. Alice adds the desired PerformanceProfile manifest for workload partitioning
+   to the `openshift` folder that was generated by the installer.
+4. Alice then creates the cluster via the installer.
+5. The installer will apply the manifests and during the bootstrapping process
+   the MCO will generate the default configurations for workload partitioning
+   based off of the `Infrastructure.Status`, and restarts the nodes.
+6. After the cluster is up the NTO will then generate the machine configurations
+   using the information provided in the PerformanceProfile manifest.
+7. The MCO applies the updated workload partitioning configurations and restarts
+   the relevant nodes.
+8. Alice will now have a cluster that has been setup with workload partitioning
+   and the desired workloads pinned to the specified CPUSet in the
+   PerformanceProfile.
+
+Applying CPU Partitioning Size Change
+
+```mermaid
+sequenceDiagram
+Alice->>API: Apply PerformanceProfile Manifest
+API-->>NTO: Apply
+loop Generate
+    NTO->>NTO: Machine Configs
+end
+NTO-->>MCO: Apply Machine Configs
+MCO-->>Node: Configure node
+loop Apply
+    Node->>Node: Set kubelet config
+    Node->>Node: Set crio config
+    Node->>Node: Kubelet advertises cores
+end
+Node-->>MCO: Finished Restart
+MCO-->>NTO: Machine Manifests Applied
+NTO-->>API: PerformanceProfile Applied
+```
+
+- **Alice** is a human user who creates an Openshift cluster.
+- **Installer** is assisted installer that applies the user manifest.
+- **NTO** is the node tuning operator.
+- **MCO** is the machine config operator.
+- **Node** is the kubernetes node.
+
+1. Alice sits down and applies the desired PerformanceProfile with the selected
+   workloads to pin.
+2. The NTO will generate the appropriate machine configs that include the
+   Kubelet config and the CRIO config and apply the machine configs.
+3. Once the MCO applies the configs, the node is restarted and the cluster has
+   been updated with the desired workload pinning.
+4. Alice will now have a cluster that has been setup with workload pinning.
+
+#### Variation [optional]
+
+##### E2E Workflow deployment
+
+This section outlines an end-to-end workflow for deploying a cluster with
+workload partitioning enabled and how pods are correctly scheduled to run on the
+management CPU pool.
+
+> Note: Items in bold are modifications/additions from the previous enhancement,
+> [Management Workload Partitioning](management-workload-partitioning.md)
+
+1. User sits down at their computer.
+2. **The user creates a `PerformanceProfile` resource with the desired
+   `isolated` and `reserved` CPUSet with the `cpu.workloads[Infrastructure]`
+   added to the enum list.**
+3. **Alice updates the `Infrastructure` CR status to denote that workload
+   partitioning is turned on.**
+4. The user runs the installer to create the standard manifests, adds their
+   extra manifests from steps 2, then creates the cluster.
```

Comment:
Steps 3 and 4 seem out of order here. The Infrastructure CR doesn't exist until `create manifests` is run, right?

Reply:
Thanks Doug, cleaned that up a bit to reflect some of the recent changes.